libcob/move.c

```diff
@@ -309,6 +309,7 @@ cob_move_alphanum_to_display (cob_field *f1, cob_field *f2)
 	const unsigned char	*e2 = s2 + COB_FIELD_SIZE (f2);
 	const unsigned char	dec_pt = COB_MODULE_PTR->decimal_point;
 	const unsigned char	num_sep = COB_MODULE_PTR->numeric_separator;
+	unsigned char	last;
```

Comment:
What do we need that for? The called function doesn't change the data, does it?
(numeric.c (cob_decimal_set_display) may change that, so that's a different thing)

Reply:
Since this function (indirectly) calls `cob_real_get_sign`, which alters the byte holding the sign (always trailing when normalizing), we have to save/restore it (or maybe we could use `cob_put_sign`).

Reply:
Yeah, probably. I'll dive further into this.